C_image_processing/filters/dark_filter.c

```diff
@@ -19,55 +20,45 @@ int dark_filter(const char *inputFile, const char *outputFile)
         return 1;
     }
 
-    unsigned char headerInfo[54];
-    unsigned char colorTable[1024];
-
-    // Read header info of image
-    for (int i = 0; i < 54; i++)
+    // read header info of image
+    for (i = 0; i < 54; i++)
     {
-        headerInfo[i] = getc(fileIn);
+        byte[i] = getc(fileIn);
```

Comment:
Changing it back to Byte would require an indication to what it represents at least a byte of what?
As if you look at it getting bigger, what would line 30 represent " fwrite(byte, sizeof(unsigned char), 54, fileOut); ", which requires going back in code in theory, without the prescribed comment to what the function does. The code itself should have minimal Comments, any place where possible add comments inside the code, apart from as I understand byte being according to documentation.

Reply:
Changed it to `headerInfo` from `byte`

Reply:
It might possibly need slightly a better name in the future, but is good for now!